.github/workflows/tests-l4lb-v1.10.yaml

```
@@ -172,6 +172,18 @@ jobs:
         run: |
           ssh default "sudo /bin/sh -c 'cd /vagrant/test/l4lb && ./test.sh ${{ github.repository_owner}} ${{ steps.vars.outputs.sha }}'"
 
+      - name: Fetch cilium-sysdump
+        if: ${{ failure() }}
```

Suggestion: we can use `!success()` rather than `failure()` in order to also fetch sysdump on cancelled runs. This would also be consistent with how we currently fetch sysdumps on other workflows :)
```suggestion
        if: ${{ !success() }}
```